target/linux/mpc85xx/image/p1020.mk

```diff
@@ -124,3 +124,21 @@ define Device/ocedo_panda
   DEVICE_COMPAT_MESSAGE := Config cannot be migrated from swconfig to DSA
 endef
 TARGET_DEVICES += ocedo_panda
+
+define Device/watchguard_t30-w
```

Comment:
Why single core device goes to p1020 instead p1010?

Reply:
I wish I could remember why exactly, but my vague memory tells me that the dev board that the device was based off was the p1020rdb, hence why I chose that target.

But, given that this is really only a p1011, I can move it over to the p1010 target.

Reply:
Also, looks like p1011 and p1020 are pin compatible, vs p1010 and p1014 are pin compatible... A bit ambiguous, but I will follow whatever you guys want

Reply:
At this moment one core devices was placed into p1010 and dual core into p1020.

Reply:
Please keep in mind that for later T50 support (which is the same board - see multiple mentions in this PR), a common DTSI could be used if the T30 remains in P1020.

Reply:
Perhaps there's a hint in the default uBoot environment as to which basis the vendor was using for the device. For example, on my T10 and T15, there are uBoot variables "fdtfile=p1010rdb.dtb" and "hostname=P1010RDB". Haven't checked my T50 yet (unopened...)

Reply:
Maybe we should make it based on the series, rather than the dual core nature?  p10xx and p20xx ?  Or should it be based on the chip layout ? Or rename to something like qoriq_p1, qoriq_p2 ?